pkg/forwarder/forwarder_health.go

```diff
@@ -26,18 +26,20 @@ const (
 )
 
 var (
-	apiKeyStatusUnknown = expvar.String{}
-	apiKeyInvalid       = expvar.String{}
-	apiKeyValid         = expvar.String{}
-	apiKeyFake          = expvar.String{}
+	apiKeyEndpointUnreachable = expvar.String{}
+	apiKeyStatusUnknown       = expvar.String{}
+	apiKeyInvalid             = expvar.String{}
+	apiKeyValid               = expvar.String{}
+	apiKeyFake                = expvar.String{}
 
 	validateAPIKeyTimeout = 10 * time.Second
 
 	apiKeyStatus = expvar.Map{}
 )
 
 func init() {
-	apiKeyStatusUnknown.Set("Unable to validate API Key")
+	apiKeyEndpointUnreachable.Set("Unable to reach the API Key validation endpoint")
+	apiKeyStatusUnknown.Set("Unexpected response code from the API Key validation endpoint")
```

Comment:
I would maybe rename this expvar to something like `apiKeyValidationError`, I feel like the current(old) name is confusing.

Reply:
Yeah, I agree with you. What do you thing about `apiKeyUnexpectedStautsCode` ?

Quite verbose but consistent with the expvar purpose.